katello/katello/backup.rb

```diff
@@ -125,6 +125,13 @@ def configure_configs
       backup_configs + feature_configs
     end
 
+    def excluded_configs
+      excluded_configs = []
+      excluded_configs.push("--exclude /var/www/html/pub/isos*") if Dir.exist?('/var/www/html/pub/isos')
+      excluded_configs.push("--exclude /var/www/html/pub/exports*") if Dir.exist?('/var/www/html/pub/exports')
```

Comment:
These `if Dir.exist?` are completely pointless. `--exclude` matches a pattern, if a pattern isn't found it doesn't matter.